src/python/model/README.md

```diff
@@ -0,0 +1,68 @@
+# Qodana imitation model 
+
+## Architecture 
```

Comment:
Please, add a description block before the Architecture block to describe what this model does in general